addon-test-support/properties/collection.js

```diff
@@ -20,6 +20,8 @@ import { collection as legacyCollection } from './collection/legacy';
  * - `forEach()` - Runs a function for each item in the collection
  * - `map()` - maps over the elements of the collection
  * - `mapBy()` - maps over the elements of the collecton by the specified property
+ * - `find()` - finds first item of the array with assert by specified function
+ * - `findBy()` - finds first item of the array with assert by property
```

Comment:
These are incorrect. This PR implements `findOne` and `findOneBy` but the docs here say it is `find` and `findBy`.

Also, Ember devs are used to arrays providing a `find` and `findBy` so it was astonishing to find this PR where the names were changed to `findOne` and `findOneBy`.

Reply:
For reference because this is hidden and differed from common interfaces I'm used to I will be using this syntax instead: `[...page.myCollection].findBy('foo', 'bar')`

Reply:
@sukima I'm sorry that you've faced that documentation issue. The docs should be fixed for sure.

> Also, Ember devs are used to arrays providing a find and findBy so it was astonishing to find this PR where the names were changed to findOne and findOneBy.

these methods have `One` for a reason. In comparison with a standard `find(` it does not look just for the first item in a list, but it ensures that the found item is the only one matching the filter condition, which is more determenistic and useful in tests.